tests/unittest/_torch/modules/mamba/benchmark_replay_selective_state_update.py

```
@@ -0,0 +1,807 @@
+
+
+# Use fast import by default; --full-import parsed later but we need the
+# functions at module level.  Check sys.argv early.
+if "--full-import" in sys.argv:
+    replay_selective_state_update, selective_state_update, causal_conv1d_update = \
+        _import_mamba_kernels_full()
+else:
+    try:
+        replay_selective_state_update, selective_state_update, causal_conv1d_update = \
+            _import_mamba_kernels_fast()
+    except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)